### PR TITLE
Addressed code review comments for schema and filter lib

### PR DESCRIPTION
**Pull request by pkharat**

### CODE REVIEW

File: gbpservice/neutron/tests/unit/nfp/configurator/lib/test_filter.py [hunk 1]
```diff
@@ -1,58 +1,94 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
```
Comment (pkharat) [inline gbpservice/neutron/tests/unit/nfp/configurator/lib/test_filter.py]: Suresh needs to remove all the ssl vpn related test cases.

File: gbpservice/neutron/tests/unit/nfp/configurator/lib/test_filter.py [hunk 2]
```diff
@@ -107,12 +144,13 @@ def test_get_logical_device(self):
...
     def test_get_vpn_servicecontext_ipsec_service_type(self):
-
+        """Test get_vpn_servicecontext() of data_filter.py
+           based on ipsec service type
+        """
         service_info = self._test_get_vpn_info()
         self.context['service_info'] = service_info
         retval = self.filter_obj.get_vpn_servicecontext(
                     self.context,
-                    constants.SERVICE_TYPE_IPSEC,
```
Comment (njagadish) [inline gbpservice/neutron/tests/unit/nfp/configurator/lib/test_filter.py]: Did we remove this as we don't need to differentiate between IPSEC and SSL now?

File: gbpservice/nfp/configurator/lib/schema.py [hunk 3]
```diff
@@ -1,17 +1,71 @@
...
 
+
+""" Request data schema.
+"""
+
 request_data = {'info': {},
                 'config': []
                 }
 
-request_data_info = {'version': "",
```
Comment (njagadish) [inline gbpservice/nfp/configurator/lib/schema.py]: We not validating the request_data_info for *aaS API. Please add that.

**Comment by njagadish** [comment]
Changes look fine. @pkharat - Can you raise the same pull request on nfp-integration also?

**Pull request merged**